apps/openmw/mwmechanics/character.hpp

```diff
@@ -152,6 +152,7 @@ struct WeaponInfo;
 class CharacterController : public MWRender::Animation::TextKeyListener
 {
     MWWorld::Ptr mPtr;
+    MWWorld::Ptr mPreviousWeapon;
```

Comment:
I think mWeapon would be a better name because this is the up to date weapon more often than it isn't.